src/app/dashboard/visit-summary/visit-summary.component.scss

```diff
@@ -754,13 +754,17 @@ th.mat-header-cell:last-child {
     align-items: flex-start;
   }
 
-  .intel-accordion-con .mat-expansion-panel .intel-accordion-title h6 {
-    font-size: 14px;
+  .intel-accordion-title{
+    flex-wrap: nowrap !important;
   }
 
-  .intel-accordion-con .mat-expansion-panel .intel-accordion-title img {
-    width: 30px;
-  }
+  // .intel-accordion-con .mat-expansion-panel .intel-accordion-title h6 {
+  //   font-size: 14px;
+  // }
```

Comment:
Is this commented code required?

Reply:
Previously that was added for PWA app, but as per QA raised issue regarding that so removed 
But in feature if need to revert as that was added previously for specific reason so i just commented the code

Reply:
If you want certain commented code please add comments above the code, approving it for now; please take care moving forward.